src/sentry/db/models/fields/foreignkey.py

```diff
@@ -1,11 +1,16 @@
 from __future__ import absolute_import
 
+from django.db import models
 from django.db.models import ForeignKey
 
 __all__ = ("FlexibleForeignKey",)
 
 
 class FlexibleForeignKey(ForeignKey):
+    def __init__(self, *args, **kwargs):
+        kwargs.setdefault("on_delete", models.CASCADE)
```

Comment:
Damn, A+.

Reply:
Damn, I didn't think of doing this, good idea. 

I'm not totally sure about handling this way, I might think about it over the weekend. One of the reasons Django made `on_cascade` explicitly required is that that they want people to actually think about how their relationships cascade, rather than defaulting to delete. It might be worth forcing that here, but maybe we don't care...

Reply:
So... I have a local branch with all the `on_delete=models.CASCADE` parameters set, so if need be we can switch to that solution pretty easily.

I think I like this solution more cause I still think the majority of the time we'll want `CASCADES` anyways. But I'm open to either method.